File: front/service/src/main/java/com/coinhunt/front/data/Statistics.java
package com.coinhunt.front.data;

import java.util.Map;
import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class Statistics {

	private final int numberOfGames;

	private final Map<Difficulty, Integer> bestTimesInMilliseconds;

	private final Map<Difficulty, Integer> averageTimesInMilliseconds;
}

File: games/src/main/kotlin/com/coinhunt/games/persistence/domain/documents/Statistics.kt
package com.coinhunt.games.persistence.domain.documents

import com.coinhunt.games.persistence.domain.components.Difficulty
import org.bson.types.ObjectId
import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document
data class Statistics(
    val numberOfGames: Long,
    val bestTimesInMilliseconds: Map<Difficulty, Long>,
    val averageTimesInMilliseconds: Map<Difficulty, Long>,
    @Id val id: ObjectId? = ObjectId.get()
)
